addons/company_loan_tracker/reports/report_company_loan.py

```
@@ -0,0 +1,116 @@
+            loans = Loan.search([])
+
+        # map selection loan_type -> label
+        selection_map = dict(Loan._fields['loan_type'].selection)
+        # thứ tự hiển thị các nhóm
+        order_key = ['short_term', 'medium_term', 'personal']
+
+        grouped_loans = []
+
+        for lt in order_key:
+            # tất cả khoản vay thuộc loại này
+            lt_records = loans.filtered(lambda l: l.loan_type == lt)
```

Ruff (0.14.2)

Ambiguous variable name: `l`

(E741)